src/app/modules/activities-management/store/activity-management.effects.ts

```diff
@@ -7,6 +7,7 @@ import { catchError, map, mergeMap } from 'rxjs/operators';
 import * as actions from './activity-management.actions';
 import { Activity } from './../../shared/models/activity.model';
 import { ActivityService } from './../services/activity.service';
+import { ActivityManagementActionTypes, DeleteActivitySuccess } from './activity-management.actions';
```

Comment:
you don't need this import because we are importing all action in line 7. please remove it.

Reply:
done